artifacts/manifests/podspec.yaml

```diff
@@ -14,7 +14,7 @@ initContainers:
       memory: 200Mi
     limits:
       cpu: 2
-      memory: 1.6Gi
+      memory: 1.5Gi
```

Comment:
I dont think we should reduce the memory limit. One option would be to add the value in `Mi` as suggested above.

I tried searching why the value `1.6Gi` was chosen but couldn't find a reason for it. We can also increase the limit of the initcontainer to be the same as the main container. WDYT. ?

Reply:
Going with `1639Mi`. Adding the change in [Makefile](https://github.com/kubernetes/publishing-bot/blob/bb90a8b609b4fdfc44fb1fa7e1f74f7a2b3871f2/Makefile#L39) as well.